cypress/utils/utils.ts

```diff
@@ -783,3 +787,15 @@ export function selectWithinModal(selector: string): void {
         click(selector);
     });
 }
+
+export function selectWithin(parent, selector: string): void {
+    cy.get(parent).within(() => {
+        click(selector);
+    });
+}
+
+export function clickReact(obgType, selector: string): void {
+    cy.waitForReact(5000, '#root');
+    // cy.get(`#${selector}`, { timeout: 2000 }).next('.pf-c-switch__toggle').should('be.visible');
+    cy.react(obgType, {props: {id: selector}}).click()
```

Comment:
{id: selector} - i don't think this is generic enough
i think maybe client should pass the entire object {props: {id: selector}}

Reply:
OK, I'll try to play with it

Reply:
a react select should be something like a struct or the ComponenetName (obgType) and the filter (selector)
so you get ("x selector) and it returns the two values

Reply:
You mean custom data type and to send it? Yes, this can work, I will consider it